task/hiffy/src/lpc55.rs

```diff
@@ -354,6 +354,8 @@ fn gpio_reset(
 pub(crate) static HIFFY_FUNCS: &[Function] = &[
     crate::common::sleep,
     crate::common::send,
+    crate::common::send_lease_read,
```

Comment:
This needs to be added to the `Functions` enum as well; as it stands, LPC55 Humliity-side consumers will call the wrong function.  And as long as this change is being made: it should also be added to the G0 array and enum.